doc/_static/theme_overrides.css

```diff
@@ -20,5 +20,29 @@
     .option {
         width: 1px !important;
     }
+
+
+    .hide {
+        display: none;
+    }
+    .hide + label ~ div {
+        display: none;
+    }
+    .hide + label {
+        display: block;
+        font-size: 1em;
+        font-weight: bold;
+        padding: 0.5em 1.5em;
+        text-shadow: 2px 2px lightgrey;
+    }
+    .hide:checked + label + div {
+        display: block;
+    }
+    .hide + label::after {
+        content: " ▶";
+    }
+    .hide:checked + label::after {
+        content: " ▼";
```

Comment:
The "▼" character makes label jump up and down depending on the collapse status.
Please fix this issue.